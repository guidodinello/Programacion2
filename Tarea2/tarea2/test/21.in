# Implementar concatenar
concatenar 0
concatenar 3 (5,3.4) (8,9.4) (7,2.6)
imprimirCadena
insertarSegmentoDespues 3 (5,3.4) (8,9.4) (7,2.6)
concatenar 0
concatenar 3 (5,3.4) (8,9.4) (7,2.6)
imprimirCadena
Fin
