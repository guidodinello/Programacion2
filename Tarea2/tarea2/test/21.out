1>#  Implementar concatenar.
2>
3>(5,3.40)(8,9.40)(7,2.60)
4>
5>Segmento insertado despuÃ©s de loc.
6>(5,3.40)(8,9.40)(7,2.60)
7>(5,3.40)(8,9.40)(7,2.60)(5,3.40)(8,9.40)(7,2.60)
8>(5,3.40)(8,9.40)(7,2.60)
9>Fin.
